src/Common/config.h.in

```diff
@@ -52,3 +52,4 @@
 #cmakedefine01 USE_ODBC
 #cmakedefine01 USE_BORINGSSL
 #cmakedefine01 USE_BLAKE3
+#cmakedefine01 ARCH_PPC64LE
```

Comment:
We don't have this for other architectures - either add all of them or remove this.

Reply:
It's done. Removed the macro and used the compiler provided macro.